Hazel/src/Hazel/Events/MouseEvent.h

```diff
@@ -52,11 +52,11 @@ namespace Hazel {
 	class MouseButtonEvent : public Event
 	{
 	public:
-		inline MouseCode GetMouseButton() const { return m_Button; }
+		[[nodiscard]] inline MouseCode GetMouseButton() const { return m_Button; }
```

Comment:
I've said it some many times before, we don't need `inline` if we're inside a class...